Relatório do Trabalho Final - Caio e Isaac

CaioMFRodrigues opened the pull request:
Done.

caioflexa committed:
Relatório do Trabalho Final - Caio e Isaac

Done.

filipesaraiva commented on trabalho-final/Dupla_Caio_Isaac:
```diff
@@ -0,0 +1,12 @@
+- Trabalho em Duplas: [https://github.com/gustavopinto/tesl/blob/master/duplas/Dupla_Caio_Isaac](#)
+
+ ## Adicionar grande funcionalidade
+Link: [https://github.com/scikit-learn/scikit-learn/pull/14239](#)
```
Colocar um bullet (`*` ou `-`).

Escrever uma frase sucinta descrevendo o que foi implementado. Veja o modelo para um exemplo.

filipesaraiva commented on trabalho-final/Dupla_Caio_Isaac:
```diff
@@ -0,0 +1,12 @@
+Link: [https://github.com/scikit-learn/scikit-learn/pull/14239](#)
+
+ ## Estudo original com o artigo sobre software livre
+Enviado por email.
```
Colocar um bullet (`*` ou `-`).

caioflexa committed:
Correção quanto ao bullut mais descrição

filipesaraiva commented on trabalho-final/Dupla_Caio_Isaac:
```diff
@@ -0,0 +1,12 @@
+- Nome completo: Caio Marcos Flexa Rodrigues/ Isaac Souza Elgrably
+- Nível: Pós-Graduação/ Pós-Graduação
+- Matrícula: 201820080007/ 201820080008
+- Trabalhos em dupla: [https://github.com/gustavopinto/tesl/blob/master/duplas/Dupla_Caio_Isaac](#)
```
corrigir os links para o formato markdown

filipesaraiva commented on trabalho-final/Dupla_Caio_Isaac:
```diff
@@ -0,0 +1,12 @@
+- Trabalhos em dupla: [https://github.com/gustavopinto/tesl/blob/master/duplas/Dupla_Caio_Isaac](#)
+
+## Adicionar grande funcionalidade
+- Inclusão de um novo algoritmo de embedding - redução da dimensionalidade de dados em alta dimensão - na biblioteca de machine learning Scikit-learn ([https://github.com/scikit-learn/scikit-learn/pull/14239](#)).
```
corrigir os links para o formato markdown

filipesaraiva reviewed (changes requested):
Adicionar ao nome do arquivo a extensão `.md`

caioflexa committed:
Correção de extensão e links

Correções atendidas:
- Adicionar ao nome do arquivo a extensão .md
- corrigir os links para o formato markdown

filipesaraiva commented:
Pronto pessoal, agora façam o PR para o relatório final da turma.
